test/extensions/filters/http/ext_proc/client_test.cc

```diff
@@ -92,7 +92,7 @@ TEST_F(ExtProcStreamTest, OpenCloseStream) {
 TEST_F(ExtProcStreamTest, SendToStream) {
   auto stream = client_->start(*this, 200ms);
   // Send something and ensure that we get it. Doesn't really matter what.
-  EXPECT_CALL(stream_, sendMessageRaw_(_, false)).Times(1);
+  EXPECT_CALL(stream_, sendMessageRaw_(_, false));
```

Comment:
Lots more of these in this PR, but I actually think I should just add a format check for this and stop harassing people :)